include/Reader/reader.h

```diff
@@ -3241,6 +3251,13 @@ class ReaderBase {
   virtual bool arrayGet(CORINFO_SIG_INFO *Sig, IRNode **RetVal) = 0;
   virtual bool arraySet(CORINFO_SIG_INFO *Sig) = 0;
 
+  /// \brief Check whether structs are represented by pointers on the operand
+  /// stack.
+  ///
+  /// \returns true iff structs are represented by pointers on the operand
+  /// stack.
+  virtual bool structsAreRepresentedByPointers() = 0;
```

Comment:
storeObj is called not only from GenIR but also from ReaderBase directly. So ReaderBase needs some way to ask GenIR about how it should interpret the node representing struct that it pops from the stack. Do you have a suggestion on how this can be done cleaner?